sdk/go/pulumi-language-go/language_test.go

```diff
@@ -172,6 +172,7 @@ func runTestingHost(t *testing.T) (string, testingrpc.LanguageTestClient) {
 
 // Add test names here that are expected to fail and the reason why they are failing
 var expectedFailures = map[string]string{
+	"l1-resource-keyword-overlap":         "outputs are not cast correctly from pcl to their pulumi types",                                           //nolint:lll
```

Comment:
Probably want a tracking issue for this if we don't want to fix as part of this PR